File: Java/Problem_120.java
public class Main {
    public static void main(String[] args) {
        long total = 0;
        for (long a = 3; a <= 1000; a++) {
            if ((a & 1) == 1) total += a*a - a;
            else              total += a*a - 2*a;
        }
        System.out.println(total); // 333082500
    }
}

File: Python/Problem_120.py
def solve(limit: int = 1000) -> int:
    total = 0
    for a in range(3, limit + 1):
        if a % 2 == 1:
            total += a * a - a
        else:
            total += a * a - 2 * a
    return total

if __name__ == "__main__":
    print(solve(1000))  # 333082500

